src/caffeine/assert/assert_s.f90

```diff
@@ -52,7 +53,7 @@
 
         end if represent_diagnostics_as_string
 
-        call prif_error_stop(header // ' with diagnostic data "' // trailer // '"')
+        call prif_error_stop(logical(.false., c_bool), stop_code_char=header // ' with diagnostic data "' // trailer // '"')
```

Comment:
Note that you can use `.false._c_bool` instead of `logical(.false., c_bool)`.

Reply:
I tried that and I got a compiler error with `gfortran`. I will try it again and copy the error here if it happens again.

Reply:
Interesting. The standard says

> R725 *logical-literal-constant* **is** .TRUE. [ _ *kind-param* ]
>                                               **or** .FALSE. [ _ *kind-param* ]

Reply:
Actually, no compiler error. I think the first time I tried that syntax here, I must not have had the `c_bool` `use-stmt` necessary and the compiler error ended up being a confusing one that made me think the syntax wasn't supported. I have updated all occurrence with your suggested syntax, since I also like it better.